templates/workspace_services/mlflow/template_schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/templates/workspace_services/mlflow/template_schema.json",
+  "type": "object",
+  "title": "MLflow machine learning lifecycle",
+  "description": "Deploys MLflow server to manage machine learning lifecycle.",
+  "required": [],
+  "properties": {
+    "is_exposed_externally": {
+      "$id": "#/properties/is_exposed_externally",
+      "type": "boolean",
+      "title": "Expose externally",
+      "description": "Is the Guacamole service exposed outside of the vnet"
```

Comment:
Reference to Guacamole needs amending.

Reply:
Fixed and will revert the non-MLFlow changes